docs/source/markdown/podman-commit.1.md

```diff
@@ -9,58 +9,52 @@ podman\-commit - Create new image based on the changed container
 **podman container commit** [*options*] *container* [*image*]
 
 ## DESCRIPTION
-**podman commit** creates an image based on a changed container. The author of the
-image can be set using the `--author` flag. Various image instructions can be
-configured with the `--change` flag and a commit message can be set using the
-`--message` flag. The container and its processes are paused while the image is
-committed. This minimizes the likelihood of data corruption when creating the new
-image. If this is not desired, the `--pause` flag can be set to false. When the commit
-is complete, Podman will print out the ID of the new image.
+**podman commit** creates an image based on a changed *container*. The author of the image can be set using the **--author** OPTION. Various image instructions can be configured with the **--change** OPTION and a commit message can be set using the **--message** OPTION. The *container* and its processes are paused while the image is committed. This minimizes the likelihood of data corruption when creating the new image. If this is not desired, the **--pause** OPTION can be set to *false*. When the commit is complete, Podman will print out the ID of the new image.
 
-If *image* does not begin with a registry name component, `localhost` will be added to the name.
-If *image* is not provided, the values for the `REPOSITORY` and `TAG` values of the created image will each be set to `<none>`.
+If `image` does not begin with a registry name component, `localhost` will be added to the name.
+If `image` is not provided, the values for the `REPOSITORY` and `TAG` values of the created image will each be set to `<none>`.
 
 ## OPTIONS
 
 #### **--author**, **-a**=*author*
 
-Set the author for the committed image
+Set the author for the committed image.
 
 #### **--change**, **-c**=*instruction*
 
 Apply the following possible instructions to the created image:
 **CMD** | **ENTRYPOINT** | **ENV** | **EXPOSE** | **LABEL** | **ONBUILD** | **STOPSIGNAL** | **USER** | **VOLUME** | **WORKDIR**
 
-Can be set multiple times
+Can be set multiple times.
 
-#### **--format**, **-f**=*format*
+#### **--format**, **-f** =**oci** | *docker*
 
-Set the format of the image manifest and metadata.  The currently supported formats are _oci_ and _docker_.  If
-not specifically set, the default format used is _oci_.
+Set the format of the image manifest and metadata.  The currently supported formats are **oci** and *docker*. The default is **oci**.
 
 #### **--iidfile**=*ImageIDfile*
 
 Write the image ID to the file.
 
-#### **--include-volumes**
+#### **--include-volumes** [=**false** | *true*]
 
-Include in the committed image any volumes added to the container by the `--volume` or `--mount` options to the `podman create` and `podman run` commands.
+Include in the committed image any volumes added to the container by the **--volume** or **--mount** OPTIONS to the [**podman create**](podman-create.1.md) and [**podman run**](podman-run.1.md) commands.
```

Comment:
Sadly these [] end up showing up in man page, but I am willing to live with this.

Reply:
I found a workaround. I will just put the `**` in front and in the back of `[]`.
Example: \**[podman create]**

Reply:
This seems to work for the man pages but not for the markdown pages.